### PR TITLE
✨ Add Avatar component

### PR DESCRIPTION
**Pull request by Maurizio-F**
Build avatar component from issue #14

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "serve": "vite preview",
     "prepare": "husky install",
     "test": "tsc && prettier --check . && eslint . && stylelint \"**/*.css\"",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "start-storybook -s src/app/assets -p 6006",
```
Comment (lmachens) [inline package.json:13]: Can you describe this change? (looks good btw)

File: src/app/components/Avatar/Avatar.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import classes from './Avatar.module.css';
+
+type AvatarProps = {
+  imageSrc: string;
+};
+
+function Avatar({ imageSrc }: AvatarProps): JSX.Element {
+  return (
+    <div>
+      <img src={imageSrc} className={classes.img} />
```
Comment (roabram) [inline src/app/components/Avatar/Avatar.tsx]: is classes.img a good naming for the avatar pic?

File: package.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
     "serve": "vite preview",
     "prepare": "husky install",
     "test": "tsc && prettier --check . && eslint . && stylelint \"**/*.css\"",
-    "storybook": "start-storybook -p 6006",
```
Comment (infoucarsinan) [inline package.json:13]: A new branch for that?

File: src/app/components/Avatar/Avatar.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    </div>
```
Comment (lmachens) [inline src/app/components/Avatar/Avatar.tsx]: Is it required to have this `div`? 
Can you remove it?

File: package.json (hunk 3 above)
Comment (jujuzitroni) [inline package.json:13]: is this change related?

File: src/app/components/Avatar/Avatar.module.css [hunk 5]
```diff
@@ -0,0 +1,6 @@
+.img {
```
Comment (lmachens) [inline src/app/components/Avatar/Avatar.module.css:1]: I would prefer to call this class like the component:

```suggestion
.Avatar {
```

or 

```suggestion
.avatar {
```

**Review by fgottesmann — APPROVED** [review]
Looks good to me so far! :)

**Review by roabram — COMMENTED** [review]
Looks good! :-)

File: src/app/components/Avatar/Avatar.tsx (hunk 2 above)
Comment (lajanzen) [inline src/app/components/Avatar/Avatar.tsx]: Maybe use BEM classes for the image? For Example: Avatar__image

**Review by infoucarsinan — COMMENTED** [review]
Super ;-)

File: src/app/components/Avatar/Avatar.module.css [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
+  width: 8em;
+  height: 8em;
+  border-radius: 100%;
+  border: 3px solid var(--primary-color);
```
Comment (lmachens) [inline src/app/components/Avatar/Avatar.module.css]: I would expect that the border size is related to the width and height.

**Review by lmachens — COMMENTED** [review]
Good job! 
Can you fix the emoji in the pull request title?

**Review by jujuzitroni — COMMENTED** [review]
lgtm!

maybe include the info that this is a component :)

**Review by Dev-dfm — COMMENTED** [review]
lgtm 👍

**Review by Niklas-Schaefers — COMMENTED** [review]
lgtm

**Review by SGierszewski — COMMENTED** [review]
Nothing more to add, good job! 👍

**Review by tmh9x — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (Maurizio-F) [reply package.json:13]: We need this to show our pics in storybook.

Comment (Maurizio-F) [reply package.json:13]: Yes, it is. ;)

File: src/app/components/Avatar/Avatar.tsx (hunk 2 above)
Comment (lmachens) [reply src/app/components/Avatar/Avatar.tsx]: There is a good article linked in:
https://github.com/lmachens/photoplay/pull/26#discussion_r639660707

File: package.json (hunk 1 above)
Comment (Maurizio-F) [reply package.json:13]: Thx ;)  that’s for configure our assets directory.

File: src/app/components/Avatar/Avatar.tsx (hunk 2 above)
Comment (Maurizio-F) [reply src/app/components/Avatar/Avatar.tsx]: We have changed this to avatar ;)

Comment (Maurizio-F) [reply src/app/components/Avatar/Avatar.tsx]: Good idea ;) thx

File: src/app/components/Avatar/Avatar.tsx (hunk 4 above)
Comment (Maurizio-F) [reply src/app/components/Avatar/Avatar.tsx]: No, we can remove it.

**Review by lmachens — APPROVED** [review]
[]~(￣▽￣)~*
